gradio/cli/commands/components/dev.py

```diff
@@ -63,6 +63,18 @@ def _dev(
         "gradio", gradio_path, cli_arg_name="--gradio-path"
     )
 
+    gradio_node_path = subprocess.Popen(
```

Comment:
I think `subprocess.call` is better suited here. It will block until the command finishes running and the stdout will already be parsed as a string (no need for `readline`).